doc/guides/naming_conventions/index.rst

```diff
@@ -0,0 +1,81 @@
+Naming Conventions
+##################
+
+Functions and Procedures
+************************
+
+#. No procedure shall have a name that is a keyword of any standard version of
```

Comment:
What is the difference between a function and procedure in C?

Reply:
The original document seems to use them interchangeably.

Reply:
just google for 'function vs procedure'... I changed this now to only talk about functions.